mos-platform/rp6502/clock.h

```diff
@@ -0,0 +1,16 @@
+#ifndef _CLOCK_H
+#define _CLOCK_H
+
+struct timespec {
```

Comment:
This will definitely end up conflicting with `<time.h>`, so if you're up for it, this seems like a good time to establish a minimum `<time.h>` in `common` with just this stuff in it.